edgelet/iotedge/src/config/apply.rs

```diff
@@ -205,6 +222,28 @@ fn execute_inner(
         ],
     });
 
+    let preloaded_master_encryption_key_bytes =
```

Comment:
for clarity's sake, can you wrap this assignment in some braces?

Reply:
Done.